contracts/TokenDistributor.sol

```diff
@@ -4,15 +4,17 @@ pragma solidity 0.8.19;
 import {ITokenDistributor} from "./interfaces/ITokenDistributor.sol";
 import {IStakingRewardsV2} from "./interfaces/IStakingRewardsV2.sol";
 import {IERC20} from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {BitMaps} from "@openzeppelin/contracts/utils/structs/BitMaps.sol";
 
 contract TokenDistributor is ITokenDistributor {
+    using BitMaps for BitMaps.BitMap;
+
+    /// @dev BitMap for storing claimed epochs
+    mapping(address to => BitMaps.BitMap claimedEpochs) private _claimedEpochsBitMap;  
```

Comment:
I think @tommyrharper made these all public. Seems silly that anyone would want to view the bitmap, but private doesn't really afford any advantages here.

Reply:
I think the only advantage would be developer experience, knowing that this declaration is only needed for internal implementation and is not meant to be user-facing. I will still arrange to make this a public storage variable though.

Reply:
@jcmonte There turned out to be another reason why this needs to be either `private` or `internal`. Solidity does not allow internal or recursive types for public state variables.


![image](https://github.com/Kwenta/token/assets/4002635/63000f51-a3ea-4cfe-908e-5a25b7374332)

Reply:
https://github.com/scorpion9979/kwenta-token/blob/816214d2f9d35503309b411877728e73726c90cd/contracts/TokenDistributor.sol#L183 should suffice